lib/gems/pending/util/miq-extensions.rb

```diff
@@ -1,5 +1,3 @@
 require 'active_support/inflector'
 require 'manageiq-gems-pending'
 require 'more_core_extensions/all'
-
-Dir.glob(File.expand_path(File.join(File.dirname(__FILE__), "extensions", "*.rb"))) { |f| require_relative "extensions/#{File.basename(f, ".*")}" }
```

Comment:
This concerns me as what will load the extensions?

Reply:
make me wonder if we need this file _at all_

Reply:
I was also diving into understanding the `win32ole` gem and what files we actually need in here.

But I had to eject and remember that this gem is a utility drawer and we will get there when we get there.